src/components/Resources/ResourcePage.js

```diff
@@ -3,46 +3,110 @@ import PropTypes from 'prop-types';
-import { Grid, Breadcrumbs, Typography } from '@material-ui/core';
+import { Grid, Breadcrumbs, Typography, Chip } from '@material-ui/core';
 import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  subtitle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+});
 
 function ResourcePage({ matchProps }) {
   const [resource, setResource] = useState({});
+  const [loading, setLoading] = useState(true);
 
   useEffect(() => {
     axios
       .get('/api/v1/resources/' + matchProps.match.params.guid)
       .then(function(response) {
         // handle success
         setResource(response.data);
+        setLoading(false);
       })
       .catch(function(error) {
         // handle error
         console.log(error);
       });
-  }, [matchProps.match.params.id]);
+  }, [matchProps.match.params.guid]);
+
+  const classes = useStyles();
+
+  const {
+    title,
+    url,
+    author,
+    tags,
+    media_type: mediaType,
+    description,
+    user,
+    date_published: datePublished,
+    modified,
+    paid,
+  } = resource;
 
   return (
-    <>
-      <Grid container spacing={1}>
-        <Grid item lg={3}>
-          <PersonalMenu />
-        </Grid>
-        <Grid item lg={9}>
-          <Breadcrumbs
-            separator={<NavigateNextIcon fontSize="small" />}
-            aria-label="breadcrumb"
-          >
-            <Link color="inherit" to="/resources">
-              Resources
-            </Link>
-            <Typography color="textPrimary">{resource.title}</Typography>
-          </Breadcrumbs>
-          <h2>{resource.title}</h2>
-          <pre>{JSON.stringify(resource, 0, 2)}</pre>
-        </Grid>
+    <Grid container spacing={1}>
+      <Grid item lg={3}>
+        <PersonalMenu />
+      </Grid>
+      <Grid item lg={9}>
+        <Breadcrumbs
+          separator={<NavigateNextIcon fontSize="small" />}
+          aria-label="breadcrumb"
+        >
+          <Link color="inherit" to="/resources">
+            Resources
+          </Link>
+          <Typography color="textPrimary">{title}</Typography>
+        </Breadcrumbs>
+        {loading ? (
+          <p>Loading...</p>
+        ) : (
+          <>
+            <Typography variant="h2" gutterBottom>
+              {title}
+            </Typography>
+            <div className={classes.subtitle}>
+              <Typography variant="subtitle1" gutterBottom>
+                <a href={url} target="_blank" rel="noopener noreferrer">
+                  {url}
+                </a>
+              </Typography>
+              <Typography variant=" this subtitle1" gutterBottom>
+                <strong>Author:</strong> {author}
+              </Typography>
+            </div>
+            {tags &&
+              tags.map(tag => {
+                return <Chip key={tag.slug} label={tag.name} />;
+              })}
+
+            <div>
+              <br></br>
```

Comment:
Prefer to use margins here instead of `<br>`. Does Material have any utility classes?

Reply:
Forgive me, I'm 'new' enough I'm not sure what you mean by margins in this case--because general whitespace shouldn't make a difference (IIRC).

Reply:
No problem - it's generally not a very good practice to use `<br>` for spacing. (see: https://html.com/tags/br/#:~:text=on%20different%20lines.-,Don't%20abuse%20line%20breaks,intrinsic%20part%20of%20the%20content.) 

Semantically speaking, it's best to use it within text. Margins or Paddings would be better used for this context https://material-ui.com/system/spacing/

Reply:
That makes a lot more sense, I'll play around with it.

Reply:
8813690 now includes a fix for this using the <Box> component from MaterialUI. Not sure if this is the cleanest solution, but eager to hear thoughts and preferences.